src/main/java/com/virtuslab/using_directives/custom/Parser.java

```diff
@@ -112,13 +112,9 @@ UsingDefs usingDirectives() {
   UsingDef usingDirective() {
     if (isValidUsingDirectiveStart(in.td.token, context.getSettings())) {
       int offset = offset(in.td.offset);
-
-      UsingDirectiveSyntax syntax = UsingDirectiveSyntax.Using;
-      if (in.td.token == Tokens.ATREQUIRE) syntax = UsingDirectiveSyntax.AtRequire;
-      else if (in.td.token == Tokens.REQUIRE) syntax = UsingDirectiveSyntax.Require;
-
       in.nextToken();
-      return new UsingDef(settings(), syntax, source.getPositionFromOffset(offset));
+      return new UsingDef(
+          settings(), UsingDirectiveSyntax.Using, source.getPositionFromOffset(offset));
```

Comment:
I think there is no point in keeping the `UsingDirectiveSyntax` as enum and passing it as there is only one possible value now.

Reply:
Yep, removed.